Jenkinsfile

```diff
@@ -15,11 +15,18 @@ elifeLibrary {
                 sh "make ci-clean"
             }
         }
-    }
 
-    elifeMainlineOnly {
-        stage 'Merge to master', {
-            elifeGitMoveToBranch commit, 'master'
+        elifeMainlineOnly {
+            stage 'Merge to master', {
+                elifeGitMoveToBranch commit, 'master'
+            }
+
+            stage 'Push unstable image', {
+                def image = DockerImage.elifesciences(this, 'sciencebeam-gym', commit)
+                def unstable_image = image.addSuffixAndTag('_unstable', commit)
+                unstable_image.tag('latest').push()
```

Comment:
I would say push the `commit` tag first and then `latest`, but I'm sure there's other places where we do it in this order. In any case :+1: on the standard pipeline for pushing the unstable image.